client/python/lib/ovmsclient/tfs_compat/grpc/serving_client.py

```diff
@@ -14,11 +14,22 @@
 # limitations under the License.
 #
 
+import grpc
+from validators import ipv4, domain
+import os
+import OpenSSL.crypto
```

Comment:
Do we want such large library dependency just for validation? Let's discuss this in next sync.